### PR TITLE
Rewrite community values 

### LINKED ISSUE #5408: Rewrite the community values statement
## Description

The [community values statement]( https://git.k8s.io/community/values.md) needs a rewrite.

## Example

Consider the first sentence:

> Kubernetes Community culture is frequently cited as a substantial contributor to the meteoric rise of this Open Source project.

- Uses evocative rather than descriptive language
- Uses passive voice
- Makes an uncited claim

This could be rewritten:
> The culture of Kubernetes contributes substantially to the project's success.

This suggested edit:
- Uses active voice
- Uses descriptive rather than evocative language
- Makes a verifiable claim

## Other improvements

Other grammatical elements also need revision. (For example, commas require review.)

## Out of scope

Changing community values is out of scope for this issue.

## Required skills

This issue requires advanced written English skills, but would otherwise make a good first issue.

For more information, see the [Kubernetes documentation style guide](https://kubernetes.io/docs/contribute/style/style-guide/).

/sig docs
/area community-management
/good-first-issue

### PR DESCRIPTION
**Pull request by sladyn98**
This pull request aims to rewrite the community values in terms of improving the grammar, active voice statements and commas. 

**Which issue(s) this PR fixes**:

Fixes #5408

### CODE REVIEW

File: values.md
Comment (reylejano) [inline values.md:19]:
```suggestion
Broadly successful and useful technologies require different perspectives and skill sets, which can only be heard in a welcoming and respectful environment.  Community membership is a privilege, not a right.  Community Leadership is earned through effort, scope, quality, quantity, and duration of contributions.  Our community shows respect for the time and effort put into a discussion, regardless of where a contributor is on their growth path.
```

**Review by reylejano — CHANGES_REQUESTED** [review]

File: values.md
Comment (reylejano) [reply values.md:3]: The original `values.md` used two spaces after each period although one space is also common.
I suggest two spaces between "success.Below".

File: values.md
Comment (reylejano) [reply values.md:11]: There is two spaces between "project  to", I suggest to using one space.

File: values.md
Comment (reylejano) [reply values.md:19]: If `values.md` continues to use two spaces after a period then I suggest to have the spacing consistent as suggested.

The suggested change of "technologies requires" to "technologies require" is from the [#sig-docs Slack conversation](https://kubernetes.slack.com/archives/C1J0BPD2M/p1610662908010300?thread_ts=1610651040.009400&cid=C1J0BPD2M) from Slack user @adamc.

Comment (adamcooke) [reply values.md:19]: I should have just posted my suggestions here. I just was being a bit lazy on Slack! Thanks for mentioning it here.

File: values.md
Comment (jdumars) [inline values.md:3]: I believe it is important to retain the reference to Kubernetes _Community_ with regards to culture. The second part could be put into active voice: The values below are distilled from years of community evolution and drive constant improvement in the project.

File: values.md
Comment (jdumars) [inline values.md:15]: I think we can omit "that" or leave it. Changing it to "the" makes it generic whereas the original phrasing is in reference to the prior sentence. I lean towards omitting it.

File: values.md
Comment (jdumars) [inline values.md:11]: It's a nit, but single spaces after periods is now the standard. This is a habit I have to break in my own writing as well.

File: values.md
Comment (jdumars) [inline values.md:19]: Single space after period is stylistically correct.

**Review by jdumars — CHANGES_REQUESTED** [review]

File: values.md
Comment (zacharysarah) [reply values.md:3]: "are distilled from" still uses passive voice. Maybe something like:
```suggestion
Kubernetes Community culture contributes substantially to the project's success. The following values evolved over years of community evolution, pushing our project and peers toward constant improvement.
```

File: values.md
Comment (zacharysarah) [reply values.md:11]:
```suggestion
We are here as a community first. Our allegiance is to the intentional stewardship of the Kubernetes project for the benefit of all its members and users everywhere. We support working together publicly for the common goal of a vibrant interoperable ecosystem, providing an excellent experience for our users. Individuals gain status through work. Companies gain status through their commitments to support this community and fund the resources necessary for the project to operate.
```

File: values.md
Comment (zacharysarah) [inline values.md:15]:
```suggestion
Large projects have a lot of hard yet less exciting work. We value time spent automating repetitive work more highly than toil. Where work cannot be automated, our culture recognizes and rewards all types of contributions while recognizing that heroism is not sustainable.
```

File: values.md
Comment (zacharysarah) [reply values.md:19]:
```suggestion
Broadly successful and useful technologies require different perspectives and skill sets, which can only be heard in a welcoming and respectful environment. Community membership is a privilege, not a right. Community members earn leadership through effort, scope, quality, quantity, and duration of contributions. Our community respects the time and effort put into a discussion, regardless of where a contributor is on their growth path.
```

File: values.md
Comment (zacharysarah) [inline values.md:23]:
```suggestion
Openness to new ideas and studied technological evolution make Kubernetes a stronger project. Continual improvement, servant leadership, mentorship, and respect are the foundations of Kubernetes culture. Kubernetes community leaders have a duty to find, sponsor, and promote new community members. Leaders should expect to step aside. Community members should expect to step up.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@sladyn98 Good start. ✨ This PR still requires some fixes.

**Comment by zacharysarah** [comment]
Oh hey, GitHub publishes the full text of suggestions in comments as both comments _and_ a separate review. That's a bracing amount of spam. 😬

File: values.md
Comment (sladyn98) [reply values.md]: Yes done

**Comment by mrbobbytables** [comment]
/hold
For sign off from steering

File: values.md
Comment (liggitt) [inline values.md:7]: non-blocking nit: I agree the verb and pluralization should agree, but would "is the cornerstone" read better? I bumped on the plural "cornerstones".

I _think_ "Distributed $stuff is the cornerstone..." is still coherent, but I'll defer to the docs folks.

**Review by liggitt — COMMENTED** [review]
looks good overall, one non-blocking suggestion

File: values.md
Comment (zacharysarah) [reply values.md:7]: @liggitt 

> would "is the cornerstone" read better?

For what it's worth, I think neither of them read especially well. :shrug: I'd like to use different language altogether, but I wanted to scope both the originating issue and any implementation reviews conservatively.

For purposes of this PR, I think either is/singular or are/plural can work.

**Comment by nikhita** [comment]
LGTM
(steering hat on)

**Comment by liggitt** [comment]
lgtm

**Comment by mrbobbytables** [comment]
lgtm

**Review by dims — APPROVED** [review]
LGTM

**Comment by cblecker** [comment]
LGTM

**Comment by nikhita** [comment]
Cancelling the hold since we have 5 steering LGTMs. :rocket: 

/hold cancel

**Pull request merged**
